docs/application/toc_all.md

```diff
@@ -14,19 +14,22 @@
 
 ## Tutorials
 
```

Comment:
Question : We have app-filtering.md under tutorials/, too, yet this page is not published. This app-filtering.md is missing in the list. Don't we need to add it?

Reply:
Reminding previous comment.

>Question : We have app-filtering.md under tutorials/, too, yet this page is not published. This app-filtering.md is missing in the list. Don't we need to add it?

Reply:
Done.